### PR TITLE
Add a helper getter method for common settings

### PR DESCRIPTION
**Pull request by juliexxia**

### CODE REVIEW

**Review by gregestren — APPROVED** [review]

**Comment by gregestren** [comment]
I think you need OWNERS approval from someone else?

**Comment by juliexxia** [comment]
Dang, I thought I was an owner to be honest.

File: rules/common_settings.bzl
Comment (alandonovan) [inline rules/common_settings.bzl:102]: My review of your PR is another PR:
https://github.com/bazelbuild/bazel-skylib/pull/286/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R68
:)

**Review by alandonovan — CHANGES_REQUESTED** [review]

File: rules/common_settings.bzl
Comment (juliexxia) [reply rules/common_settings.bzl:102]: Not sure what the implication is here - are you saying you don't think this new content meets the criteria you laid out?

**Comment by aiuto** [comment]
ping. lets resolve this.

File: rules/common_settings.bzl
Comment (alandonovan) [reply rules/common_settings.bzl:102]: My point is that this is not a good abstraction, and we certainly wouldn't want users to declare a dependency on this repo for a function like this. It neither hides concepts from the user, nor simplifies the cognitive load. ("The doc comment is longer than the function body." 

(It doesn't even save the user keystrokes: the function's name is longer than it body!)

Comment (gregestren) [reply rules/common_settings.bzl:102]: I think @juliexxia had a newer perspective on this PR. Let's hold off more debate until she can offer her input.

**Pull request closed without merging**
